shoov/modules/custom/shoov_ci_build/shoov_ci_build.module

```diff
@@ -433,3 +438,27 @@ function shoov_ci_build_trigger_event(Message $message, $event) {
   $data = array($result[0]);
   shoov_pusher_trigger_event($wrapper->field_ci_build->og_repo->getIdentifier(), $event, $data);
 }
+
+/**
+ * Get all messages with Queue status of certain CI Build.
+ *
+ * @param $ci_build_id
+ *  CI Build node ID.
+ *
+ * @return array
+ *  Return array of messages ids.
+ */
+function shoov_ci_build_get_queue_items($ci_build_id) {
```

Comment:
This should be approached as an API function - change signature to `$node, $status ='queue'`

Also, since we have `fieldCondition` here, it might be better to set the $node's author as the account (`setMetaData`)